css/styles.css

```diff
@@ -0,0 +1,116 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.main-header a {
+  text-decoration: none;
+  color: #000;
+}
+
+.george {
+  max-width: 5%;
+}
+
+.row {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.column-full,
+.column-half {
+  width: 100%;
+}
+
+.main-header {
+  background-color: #9eff7c;
+  display: flex;
+  padding: 20px;
```

Comment:
I would use rem instead of px because it allows for scalable and accessible design!